ui/components-react/src/components-react/filter-builder/FilterBuilder.scss

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+.fb-component {
```

Comment:
We are not using CSS modules, could we make root selectors more unique? I.e. 
`.components-filterBuilder-filterBuilder { .fb-wrapper { /* ... */ } }`